system-contracts/contracts/EvmInterpreterLoop.template.yul

```diff
@@ -812,7 +811,6 @@ for { } true { } {
         offset, sp := popStackItemWithoutCheck(sp)
         size, sp := popStackItemWithoutCheck(sp)
 
-        // TODO overflow checks
         checkMemOverflowByOffset(add(offset, size), evmGasLeft)
```

Comment:
Same, we need to guarantee that `offset + size` and `destOffset + size` can't overflow

Reply:
Done!